pkg/cluster/default_manager.go

```diff
@@ -3,8 +3,20 @@ package cluster
 var defaultManager *ConfigManager
 
 func init() {
-	defaultManager = NewConfigManager()
-	err := defaultManager.Register(
+	var err error
+	defaultManager, err = NewDefaultConfigManager()
```

Comment:
why do we still need this in `init()` if it's being called in spec.go? just for checking the error?

Reply:
Discussed offline